datatypes/src/raster/tiling.rs

```diff
@@ -0,0 +1,206 @@
+use crate::primitives::{BoundingBox2D, Coordinate2D, SpatialBounded};
+
+use super::{GeoTransform, GridBoundingBox2D, GridIdx, GridIdx2D, GridShape2D, GridSize};
+
+use serde::{Deserialize, Serialize};
+
+/// The static parameters of a `TilingStrategy`
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+pub struct TilingSpecification {
+    pub origin_coordinate: Coordinate2D,
+    pub tile_size: GridShape2D,
```

Comment:
this is in pixels right? maybe make this more clear (rename/doc)

Reply:
1f6f43c